.vscode/launch.json

```diff
@@ -17,7 +17,8 @@
         "script": "watch"
       },
       "env": {
-        "SERVER_PATH": "${workspaceRoot}/server/target/"
+        "SERVER_PATH": "${workspaceRoot}/server/target/",
+        "SLICE_PATH": "${workspaceRoot}/slice/"
```

Comment:
Maybe this should be `BUILT_IN_SLICE_PATH` for clarity?

Reply:
I agree that would be a clearer name.